### ISSUE TITLE
feature request : auto zip files

### ISSUE THREAD

**Issue by mwolfaardt** [filegator/filegator#429]
Have the ability to auto zip files once they are uploaded.

**Comment by JustinLovell** [comment]
Yes, the zipping feature confuses clients as once it starts happening, we get a spinning wheel with no timeline and people think the page is stuck.
